test/import/__snapshots__/import-crd.test.ts.snap

```diff
@@ -123892,6 +123892,5645 @@ export function toJson_TenantV1Beta1SpecNetworkPoliciesItemsIngressFromPodSelect
+  if (obj === undefined) { return undefined; }
+  const result = {
+    'metadata': obj.metadata,
+    'spec': toJson_PostgresqlSpec(obj.spec),
+  };
+  // filter undefined values
+  return Object.entries(result).reduce((r, i) => (i[1] === undefined) ? r : ({ ...r, [i[0]]: i[1] }), {});
+}
+/* eslint-enable max-len, quote-props */
+
+/**
+ * @schema PostgresqlSpec
+ */
+export interface PostgresqlSpec {
+  /**
+   * @schema PostgresqlSpec#additionalVolumes
+   */
+  readonly additionalVolumes?: PostgresqlSpecAdditionalVolumes[];
+
+  /**
+   * @schema PostgresqlSpec#allowedSourceRanges
+   */
+  readonly allowedSourceRanges?: string[];
+
+  /**
+   * @schema PostgresqlSpec#clone
+   */
+  readonly clone?: PostgresqlSpecClone;
+
+  /**
+   * @schema PostgresqlSpec#connectionPooler
+   */
+  readonly connectionPooler?: PostgresqlSpecConnectionPooler;
+
+  /**
+   * @schema PostgresqlSpec#databases
+   */
+  readonly databases?: { [key: string]: string };
+
+  /**
+   * @schema PostgresqlSpec#dockerImage
+   */
+  readonly dockerImage?: string;
+
+  /**
+   * @schema PostgresqlSpec#enableConnectionPooler
+   */
+  readonly enableConnectionPooler?: boolean;
+
+  /**
+   * @schema PostgresqlSpec#enableReplicaConnectionPooler
+   */
+  readonly enableReplicaConnectionPooler?: boolean;
+
+  /**
+   * @schema PostgresqlSpec#enableLogicalBackup
+   */
+  readonly enableLogicalBackup?: boolean;
+
+  /**
+   * @schema PostgresqlSpec#enableMasterLoadBalancer
+   */
+  readonly enableMasterLoadBalancer?: boolean;
+
+  /**
+   * @schema PostgresqlSpec#enableMasterPoolerLoadBalancer
+   */
+  readonly enableMasterPoolerLoadBalancer?: boolean;
+
+  /**
+   * @schema PostgresqlSpec#enableReplicaLoadBalancer
+   */
+  readonly enableReplicaLoadBalancer?: boolean;
+
+  /**
+   * @schema PostgresqlSpec#enableReplicaPoolerLoadBalancer
+   */
+  readonly enableReplicaPoolerLoadBalancer?: boolean;
+
+  /**
+   * @schema PostgresqlSpec#enableShmVolume
+   */
+  readonly enableShmVolume?: boolean;
+
+  /**
+   * @schema PostgresqlSpec#env
+   */
+  readonly env?: any[];
+
+  /**
+   * deprecated
+   *
+   * @schema PostgresqlSpec#init_containers
+   */
+  readonly initContainers?: any[];
+
+  /**
+   * @schema PostgresqlSpec#logicalBackupSchedule
+   */
+  readonly logicalBackupSchedule?: string;
+
+  /**
+   * @schema PostgresqlSpec#maintenanceWindows
+   */
+  readonly maintenanceWindows?: string[];
+
+  /**
+   * @schema PostgresqlSpec#nodeAffinity
+   */
+  readonly nodeAffinity?: PostgresqlSpecNodeAffinity;
+
+  /**
+   * @schema PostgresqlSpec#numberOfInstances
+   */
+  readonly numberOfInstances: number;
+
+  /**
+   * @schema PostgresqlSpec#patroni
+   */
+  readonly patroni?: PostgresqlSpecPatroni;
+
+  /**
+   * @schema PostgresqlSpec#podAnnotations
+   */
+  readonly podAnnotations?: { [key: string]: string };
+
+  /**
+   * deprecated
+   *
+   * @schema PostgresqlSpec#pod_priority_class_name
+   */
+  readonly podPriorityClassName?: string;
```

Comment:
This is what we are asserting again. Only one property is emitted, like expected.